Framework/Core/Windows/Devices/Spi/SpiDevice.cs

```diff
@@ -22,38 +26,20 @@ public sealed class SpiDevice : IDisposable
         /// </summary>
         /// <param name="deviceId">The unique name of the device.</param>
         /// <param name="settings">Settings to open the device with.</param>
-        internal SpiDevice(string deviceId, SpiConnectionSettings settings)
+        internal SpiDevice( string deviceId, SpiConnectionSettings settings )

```

Comment:
Major reformatting changes should be split into separate commits.  Small ones are fine, but it's really hard to see where the functional changes are here.